apps/portal/components/copy-button.tsx

```diff
@@ -0,0 +1,65 @@
+import { Copy, Share } from "lucide-react";
+import React from "react";
+
+import { useToast } from "@recallnet/ui/hooks/use-toast";
+import { cn } from "@recallnet/ui/lib/utils";
+
+// Extend SVGProps for direct SVG element attributes
+type CopyButtonProps = {
+  value: string;
+  type?: "copy" | "share";
+  tooltip?: string;
+  successMessage?: string;
+  // We still need a wrapperClassName for the tooltip span
+  wrapperClassName?: string;
```

Comment:
There is no reason to let the caller style the wrapper, just remove this prop, we don't use it anyway.